src/main/java/data_access/InMemoryUserDataAccessObject.java

```diff
@@ -48,6 +48,16 @@ public class InMemoryUserDataAccessObject implements ClubSignupUserDataAccessInt
     private final DataStoreArrays<Student> studentArrayList = new DataStoreArrays<>();
     private final DataStoreArrays<Club> clubArrayList = new DataStoreArrays<>();
 
+    public InMemoryUserDataAccessObject() {
+        final Club club = new Club("test", "@.", "123123123", new DataStoreArrays<>(),
+                new DataStoreArrays<>());
+        final Student student = new Student("test", "@.", "123123123", new DataStoreArrays<>());
+        student.joinClub(club);
+        club.addClubMember(student);
+        clubArrayList.add(club);
+        studentArrayList.add(student);
+    }
+
```

Comment:
This will be for testing purposes. 
![image](https://github.com/user-attachments/assets/229ebba0-7895-49e4-a574-eb0a87058798)